package.json

```diff
@@ -1,49 +1,51 @@
 {
-  "name": "com.massonemechanical.fieldworker",
+  "name": "massonemechanical",
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
     "android": "react-native run-android",
     "ios": "react-native run-ios",
-    "test": "jest"
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint ."
   },
   "dependencies": {
+    "@react-native-community/async-storage": "^1.8.0",
+    "@react-native-community/netinfo": "^5.5.0",
     "flux": "^3.1.3",
     "flux-state": "^4.1.0",
     "i18next": "^12.1.0",
-    "moment": "^2.22.2",
-    "moment-timezone": "^0.5.23",
+    "moment": "^2.24.0",
+    "moment-timezone": "^0.5.27",
     "native-base": "^2.8.1",
     "pure-logger": "0.0.1",
-    "react": "16.6.1",
+    "react": "16.9.0",
     "react-i18next": "^8.3.8",
-    "react-native": "0.57.8",
+    "react-native": "0.61.5",
+    "react-native-firebase": "^5.6.0",
     "react-native-document-picker": "^2.2.0",
-    "react-native-dotenv": "^0.2.0",
-    "react-native-firebase": "^5.2.3",
     "react-native-gesture-handler": "^1.0.9",
     "react-native-image-picker": "^0.27.1",
     "react-native-iphone-x-helper": "^1.2.1",
     "react-native-linear-gradient": "^2.5.6",
     "react-native-modal-datetime-picker": "^6.1.0",
-    "react-native-offline": "^3.15.0",
+    "react-native-offline": "^5.4.0",
+    "react-native-screens": "^2.0.0-beta.6",
     "react-native-signature-capture": "^0.4.9",
-    "react-native-view-pdf": "^0.8.3",
+    "react-native-view-pdf": "^0.10.1",
     "react-navigation": "^3.0.0",
     "rxjs": "^6.3.3"
   },
   "devDependencies": {
-    "babel-eslint": "^10.0.1",
-    "babel-jest": "23.6.0",
-    "eslint": "^5.9.0",
-    "eslint-plugin-react": "^7.11.1",
-    "husky": "^1.2.0",
-    "jest": "23.6.0",
-    "lint-staged": "^8.1.0",
-    "metro-react-native-babel-preset": "0.49.2",
-    "prettier": "^1.15.2",
-    "react-test-renderer": "16.6.1"
+    "@babel/core": "^7.8.4",
```

Comment:
hay que agregarles las opciones del formateador:
https://github.com/cobuildlab/cobuildlab-web/blob/master/src/pages/development-blog/conventions-to-create-a-react-application.md#4-formatting-and-linting